test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -301,10 +301,13 @@ function prepare-log-file {
 function setup-addon-manifests {
   local -r src_dir="${KUBE_ROOT}/$2"
   local -r dst_dir="/etc/kubernetes/$1/$2"
+	local files=""
+
   if [[ ! -d "${dst_dir}" ]]; then
     mkdir -p "${dst_dir}"
   fi
-  local files=$(find "${src_dir}" -maxdepth 1 -name "*.yaml")
+
+  files=$(find "${src_dir}" -maxdepth 1 -name "*.yaml")
```

Comment:
nit: if SC2155 is going to force us to break up decls, I'd prefer to see this right next the the decl

Reply:
Can do that.

Reply:
Done.